hw/ip/otp_ctrl/doc/_index.md

```diff
@@ -813,6 +817,12 @@ The LIFE_CYCLE partition is the only partition that is modified in the field - b
 OTP words cannot be programmed twice, and doing so may damage the memory array.
 Hence the OTP controller performs a blank check and returns an error if a write operation is issued to an already programmed location.
 
+### Potential Side-Effects on Flash via Life Cycle
```

Comment:
@tjaychen @cindychip FYI: added a few words of caution, highlighting this dependence.